std-bits/base/src/main/java/org/enso/base/statistics/CountMinMax.java

```diff
@@ -1,52 +1,35 @@
 package org.enso.base.statistics;
 
-import java.util.Arrays;
-import java.util.Comparator;
-import java.util.Iterator;
-import java.util.function.BiFunction;
-import java.util.stream.Stream;
+import org.graalvm.polyglot.Value;
 
 public class CountMinMax {
```

Comment:
I'm not sure if I understand why we keep that in Java?
Seems like it should be well emulated by a collection of 4 Refs.
Although I do like that it is encapsulated in a purposeful object - this is neat. Just wondering if the object couldn't be managed by Enso? Or were Refs not performant enough here?

Reply:
Massively slower

Reply:
Oh, ok then. interesting though as in principle it should have similar characteristics. Is it worth discussing with @JaroslavTulach if we want to find out why?

Reply:
yes we will have a ticket to look at it